pycobertura/cobertura.py

```diff
@@ -362,17 +362,14 @@ def diff_line_rate(self, filename=None):
 
     def diff_missed_lines(self, filename):
         """
-        Return a list of 2-element tuples `(lineno, is_new)` for the given
-        file `filename` where `lineno` is a missed line number and `is_new`
-        indicates whether the missed line was introduced (True) or removed
-        (False).
-        """
-        line_changed = []
-        for line in self.file_source(filename):
-            if line.status is not None:
-                is_new = not (line.status)
-                line_changed.append((line.number, is_new))
-        return line_changed
+        Return a list of 2-element tuples `(lineno, True)` for uncovered lines.
+        The given file `filename` where `lineno` is a missed line number.
+        """
+        return [
+            (line.number, True)
+            for line in self.file_source(filename)
+            if line.status is False
+        ]
```

Comment:
The second item returned in the tuple helped reporters colorize added/remove lines (green/red). Although, since we're deciding that the missed column will only contain missed lines, there isn't a need anymore to return that second item in the tuple. We can actually return a scalar value of the missed line number only.

```suggestion
        return [
            line.number
            for line in self.file_source(filename)
            if line.status is False
        ]
```

Then we'd have to update the other parts of the code that depend on `diff_missed_lines()` to not treat the value as a tuple anymore.

Additionally, I don't think we need to render the `+` anymore (see `format_missed_lines()`) since we no longer need to distinguish an uncovered line from a covered one anymore. Although I think renderers should still render missed lines as red as it highlights to the user where their next coverage efforts should be put into.

Does that make sense?